aspnetcore/mvc/advanced/custom-model-binding/sample/CustomModelBindingSample/Startup.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
```

Comment:
It's a trivial example aimed at advanced users. They'll be able to quickly add that back in if needed. All that template code obscures the changes you are making.